Source/CesiumRuntime/Private/CesiumPropertyAttribute.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "CesiumPropertyAttribute.h"
 #include <CesiumGltf/PropertyAttributeView.h>
+#include "CesiumRuntime.h"
```

Comment:
Swap order so the third-party include is at the bottom:

```suggestion
#include "CesiumRuntime.h"
#include <CesiumGltf/PropertyAttributeView.h>
```